pages/1.12/cli/command-reference/dcos-cluster/index.md

```diff
@@ -32,10 +32,10 @@ dcos cluster
 
 | Command | Description |
 |---------|-------------|
-| [dcos cluster attach](/1.11/cli/command-reference/dcos-cluster/dcos-cluster-attach/)   |  Attach the CLI to a connected or linked cluster. |
-| [dcos cluster link](/1.11/cli/command-reference/dcos-cluster/dcos-cluster-link/)       |  (DC/OS Enterprise only) Link a cluster to another cluster.  |
-| [dcos cluster list](/1.11/cli/command-reference/dcos-cluster/dcos-cluster-list/)       |  List the clusters that are connected to the DC/OS CLI.  |
-| [dcos cluster remove](/1.11/cli/command-reference/dcos-cluster/dcos-cluster-remove/)   |  Remove a cluster from the DC/OS CLI configuration.   |
-| [dcos cluster rename](/1.11/cli/command-reference/dcos-cluster/dcos-cluster-rename/)   |  Rename a cluster in the DC/OS CLI configuration.  |
-| [dcos cluster setup](/1.11/cli/command-reference/dcos-cluster/dcos-cluster-setup/)     |  Connects, authenticates, and attaches the DC/OS CLI to a DC/OS cluster. Combines [`dcos config set core.dcos_url`](/1.11/cli/command-reference/dcos-config/dcos-config-set/), [`dcos auth login`](/1.11/cli/command-reference/dcos-auth/dcos-auth-login/), and `docs cluster attach`. |
-| [dcos cluster unlink](/1.11/cli/command-reference/dcos-cluster/dcos-cluster-unlink/)   |  (DC/OS Enterprise only) Unlink a cluster from another cluster. |
+| [dcos cluster attach](/1.12/cli/command-reference/dcos-cluster/dcos-cluster-attach/)   |  Attach the CLI to a connected or linked cluster. |
+| [dcos cluster link](/1.12/cli/command-reference/dcos-cluster/dcos-cluster-link/)       |  (DC/OS Enterprise only) Link a cluster to another cluster.  |
+| [dcos cluster list](/1.12/cli/command-reference/dcos-cluster/dcos-cluster-list/)       |  List the clusters that are connected to the DC/OS CLI.  |
+| [dcos cluster remove](/1.12/cli/command-reference/dcos-cluster/dcos-cluster-remove/)   |  Remove a cluster from the DC/OS CLI configuration.   |
+| [dcos cluster rename](/1.12/cli/command-reference/dcos-cluster/dcos-cluster-rename/)   |  Rename a cluster in the DC/OS CLI configuration.  |
+| [dcos cluster setup](/1.12/cli/command-reference/dcos-cluster/dcos-cluster-setup/)     |  Connects, authenticates, and attaches the DC/OS CLI to a DC/OS cluster. Combines [`dcos config set core.dcos_url`](/1.12/cli/command-reference/dcos-config/dcos-config-set/), [`dcos auth login`](/1.12/cli/command-reference/dcos-auth/dcos-auth-login/), and `docs cluster attach`. |
+| [dcos cluster unlink](/1.12/cli/command-reference/dcos-cluster/dcos-cluster-unlink/)   |  (DC/OS Enterprise only) Unlink a cluster from another cluster. |
```

Comment:
Replace (DC/OS Enterprise only) with [enterprise type="inline" size="small" /]. 

In our documentation, when we want to represent content/lines/features that are specific to Enterprise only or Open Source only then it is recommended to use tags instead of text.
Enterprise tags [enterprise type="inline" size="small" /] 
Open Source tags [oss type="inline" size="small" /]